RoboFile.php

```diff
@@ -351,6 +351,53 @@ function qaCodeSniffer($severity='errors') {
       ->run();
   }
 
+  function qaFixFile($filePath) {
+    if (substr($filePath, -4) === '.php') {
+      // fix PHPCS rules
+      return $this->collectionBuilder()
+        ->taskExec(
+          './tasks/code_sniffer/vendor/bin/phpcbf ' .
+            '--standard=./tasks/code_sniffer/MailPoet ' .
+            '--runtime-set testVersion 5.6-7.3 ' .
+            $filePath . ' -n'
```

Comment:
I don't see any `fix` argument here. Or is that the `-n`?

Reply:
Here I am using `phpcbf` instead of `phpcs` to fix the errors. The `-n` flag is to not print warnings.